nixos/modules/services/web-apps/miniflux.nix

```diff
@@ -61,11 +62,16 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions = [
+      { assertion = cfg.config.CREATE_ADMIN == 0 || cfg.adminCredentialsFile != null;
+        message = "Admin credentials must be provided if admin creation is enabled.";
```

Comment:
Please use the full option name in the message, e.g. `services.miniflux.adminCredentialsFile must be set if services.miniflux.config.CREATE_ADMIN is 0`

Reply:
Thanks for the review. Fixed.